src/ImageSearchModal.jsx

```diff
@@ -161,10 +162,12 @@ export class ImageSearchModal extends React.Component {
                     </Flex>
                 </Form>
 
-                {this.state.searchInProgress && <Bullseye><Spinner id='search-image-dialog-waiting' /></Bullseye>}
+                {this.state.searchInProgress && <EmptyStatePanel loading title={_("Searching...")} /> }
 
-                {this.state.searchFinished && !this.state.imageIdentifier == '' && <>
-                    {this.state.imageList.length == 0 && <div className="no-results"> {cockpit.format(_("No results for $0. Please retry another term."), this.state.imageIdentifier)} </div>}
+                {this.state.imageIdentifier == "" && <EmptyStatePanel title={_("No images found")} paragraph={_("Start typing to look for images.")} /> }
+
+                {this.state.searchFinished && this.state.imageIdentifier !== '' && <>
+                    {this.state.imageList.length == 0 && <EmptyStatePanel icon={ExclamationCircleIcon} title={cockpit.format(_("No results for $0. Please retry another term."), this.state.imageIdentifier)} />}
```

Comment:
Please add the Please ... part in the paragraph like you have for the hint above. Let's put the hints in the paragraph section always.

Reply:
![noreultsplease](https://user-images.githubusercontent.com/12330670/100474414-d63c0f80-30e0-11eb-901c-1a406f5b6acc.png)
![pleaaseprettyplewase](https://user-images.githubusercontent.com/12330670/100474415-d76d3c80-30e0-11eb-81a1-406aa702ffde.png)